luaui/Widgets/gui_reclaim_field_highlight.lua

```diff
@@ -408,7 +413,7 @@ local minFeatureMetal = 9 -- Tick
 local E2M = 1 / 70 -- Converter ratio
 local minDim = 100
 
-local checkFrequency = 30
+local checkFrequency = 2 * Game.gameSpeed
```

Comment:
From my in game tests this is still too much. Wrecks are reclaimed or added but values don't update for a full second and reclaim field remains on map, now linking nonexisting ghost wrecks.